go/vt/vttablet/tabletmanager/vreplication/replicator_plan.go

```diff
@@ -257,7 +258,7 @@ func (tp *TablePlan) applyBulkInsert(sqlbuffer *bytes2.Buffer, rows []*querypb.R
 		if i > 0 {
 			sqlbuffer.WriteString(", ")
 		}
-		if err := appendFromRow(tp.BulkInsertValues, sqlbuffer, tp.Fields, row, tp.FieldsToSkip); err != nil {
+		if err := tp.appendFromRow(tp.BulkInsertValues, sqlbuffer, tp.Fields, row, tp.FieldsToSkip); err != nil {
```

Comment:
If we make this change, which I'm OK with, then we don't need to pass in the other tp member values:
```
tp.appendFromRow(sqlbuffer, row)
```

Reply:
Good catch! Fixed.